src/scripts/core/contour.js

```diff
@@ -315,7 +315,7 @@
             _.each(this._extraOptions, mergeExtraOptions);
             _.each(this._visualizations, mergeDefaults);
 
-            var opt = _.extend({}, this.originalOptions);
+            var opt = _.nw.materialize(this.originalOptions, this, { skipMatch: /formatter/ });
```

Comment:
Note to self: this needs to be updated to remove param `skipMatch` and merge the original `skip` list

```
var opt = _.nw.materialize(this.originalOptions, this, { skip: [/formatter/].concat(this.originaOptions.skip) });
```